tests/e2e/macvtap_test.go

```diff
@@ -231,6 +234,16 @@ func filterPods(pods []v1.Pod, filterFunction func(v1.Pod) bool) []v1.Pod {
 	return filteredPods
 }
 
+func findMacvtapNamespace() string {
+	// Retrieve manager namespace serching for pods
+	// with label name=kubemacpool and getting the env var POD_NAMESPACE
+	pods, err := clientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{LabelSelector: "name=macvtap-cni"})
```

Comment:
Please use the constant for this - `v1.NamespaceAll` , from `k8s.io/api/core/v1` .

Plus, why check the pods ? Wouldn't the daemonset be the correct entity to look for ? (it's what we deploy).

Reply:
you're totally right. fixing.